maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -310,6 +348,37 @@ private void downloadOWAs(File targetDirectory, DistroProperties distroPropertie
 		}
 	}
 
+	private void setConfigFolder(Server server, DistroProperties distroProperties) throws MojoExecutionException {
+		if(distroProperties != null) {
+			File configDir = new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_CONFIGURATION);
+			configDir.mkdir();
+			downloadConfigs(distroProperties, configDir);
+			File configFile = new File(configDir, "referenceapplication-config.owa");
+			try {
+				if(!configFile.exists()) return;
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle

Reply:
I fixed this formatting issue.